upper_letters.rb

```diff
@@ -0,0 +1,19 @@
+puts "Digite uma frase"
+phrase = gets.chomp
+
+cont = 0
+phrase_split = phrase.split('')
+
+phrase_split.each_with_index do |char, index|
+	if char == phrase_split[index].capitalize
+		cont = cont + 1
+	end
+end
```

Comment:
Só lhe adicionando informação, você não precisa utilizar o `each_with_index` nesses contextos, porque o each já nos trás o valor do array naquela posição, através do `char` que tem ali depois do `do |char| `

```suggestion
phrase_split.each do |char|
			count++ if char == char.capitalize
end
```

Porém a ideia também seria a gente criar o nosso próprio método `upcase`.

`capitalize` ele pega um frase `"areae"` e transforma somente o primeiro elemento, o `upcase` transforma em tudo, o que nós desejamos é criar um `upcase`